Utils/Dataflow/069_upload2es/load_data.sh

```diff
@@ -35,25 +69,41 @@ load_files () {
   log "Putting data to ES"
   for INPUTFILE in $*;
   do
-    ${cmd}${INPUTFILE} || exit 3
+    [ ${cmd}${INPUTFILE} ] && echo -ne "$EOMessage" || exit 3
```

Comment:
We do not need to output EOMessage in Sink Connector stages (just as in both previous comments about this).

Reply:
And... does it work, actually? I have this error in a similar situation:
```
$ cmd="curl http://127.0.0.1:9200/_bulk?pretty --data-binary @"
$ filename="filename"
$ [ ${cmd}${filename} ] && echo 1
-bash: [: too many arguments
```